### PR TITLE
Refactor for Virtualmin 7 to simplify repos

### PR DESCRIPTION
**Pull request by swelljoe**
Consolidate all binaries into one universal (per architecture) repository.

Related to https://github.com/virtualmin/virtualmin-install/issues/65

### CODE REVIEW

File: rpmbuild/SOURCES/virtualmin.repo [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [virtualmin]
-name=RHEL/CentOS $releasever - $basearch - Virtualmin
-baseurl=http://SERIALNUMBER:LICENSEKEY@software.virtualmin.com/vm/6/centos/$releasever/$basearch/
+name=Binary RPM - $basearch - Virtualmin
```
Comment (iliajie) [inline rpmbuild/SOURCES/virtualmin.repo:2]: What about:

```
Virtualmin 7 Packages ($basearch)
```

**Comment by iliaross** [comment]
What about names, like:

```
Virtualmin 7 Binary ($basearch)
```

and 

```
Virtualmin 7 Neutral
```

**Comment by swelljoe** [comment]
Yeah, that's better. I'll update.

**Comment by iliaross** [comment]
Thanks for updating it!

Although, for some reason I don't like it to be called "Universal" .. rather "Neutral". But I guess if it happened to be historically like this then fine.

I would call it "Neutral" and it seems to go better with "x86_64" rather than "Universal".

**Comment by swelljoe** [comment]
noarch is what they call it upstream. I'll just change it to that.

**Comment by iliaross** [comment]
`noarch` is great. Let's change URL too?

**Comment by swelljoe** [comment]
Hmmm...should I go ahead and sort of make everything consistent?

e.g.:

`software.virtualmin.com/vm/7/pro/rpm/noarch`

`software.virtualmin.com/vm/7/gpl/rpm/noarch`

Since, apt has an `apt` subdir in gpl.

**Comment by iliaross** [comment]
Yeah, consistency is great!

**Pull request merged**